test/functional/fixtures/runner/browser-dialogs/before-unload-dialog-test.js

```diff
@@ -0,0 +1,34 @@
+var expect = require('chai').expect;
+
+describe('Native dialogs handling', function () {
+    it('Should succeed when expected before unload dialog appeared', function () {
+        return runTests('before-unload-dialog.test.js', 'Before unload dialog - should succeed')
+            .then(function (err) {
+                expect(err).eql('');
+            });
+    });
+
+    it('Should fail when expected before unload dialog did not appear', function () {
+        return runTests('before-unload-dialog.test.js', 'No expected before unload dialog - should be failed', { shouldFail: true })
+            .catch(function (err) {
+                var expectedError = [
+                    'Error at step "1.Click link "This page"":',
+                    ' The expected system beforeUnload dialog did not appear.'
+                ].join('');
+
+                expect(err).eql(expectedError);
+            });
+    });
+
+    it('Should fail when unexpected before unload dialog appeared', function () {
+        return runTests('before-unload-dialog.test.js', 'Unexpected before unload dialog - should be failed', { shouldFail: true })
```

Comment:
All of our server test names have the following structure: `it('Should <do smthg>')` (Please see, `test/server/*-test.js` files in our repo). Maybe we should keep this template here too?
/cc @MargaritaLoseva @inikulin